server/controller/src/main/java/ai/starwhale/mlops/schedule/k8s/K8sJobTemplate.java

```diff
@@ -49,6 +50,8 @@ public class K8sJobTemplate {
 
     private final String pipCacheHostPath;
 
+    public static final String deviceLabelName = "job.starwhale.ai/device";
```

Comment:
`public static final String`s in java are in uppercase `LABE_DEVICE_NAME`

Reply:
Done